ui/mirage/factories/sync-association.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { Factory } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  accessor: 'kv_eb4acbae',
+  secret_name: 'my-secret-1',
```

Comment:
Do we want to make this a `nested/secret` ? Since most secrets in kv are nested

Reply:
I just copied the example from the docs as the default. The way it is set up is the `secret_name` param will be used when setting an association which will be the value selected in the UI. I'm happy to update the default though to something like `my-path/my-secret-1` for example.